src/NoteField.cpp

```diff
@@ -868,7 +866,7 @@ void NoteField::DrawPrimitives()
 		ASSERT(GAMESTATE->m_pCurSong != nullptr);
 
 		const TimingData &timing = *pTiming;
-		const RageColor text_glow= RageColor(1,1,1,RageFastCos(RageTimer::GetTimeSinceStartFast()*2)/2+0.5f);
+		const RageColor text_glow= RageColor(1,1,1,(RageTimer::GetTimeSinceStartFast()));
```

Comment:
Let's keep the original math here and below

Reply:
I was literally pushing this back here as you commented it!